.github/workflows/pr-workflow.yml

```diff
@@ -38,7 +38,7 @@ jobs:
         uses: docker/setup-buildx-action@v1
   
       - name: Build Docker Image
-        run: docker build . --file ./Dockerfile
+        run: docker build . --file ./Dockerfile.local
```

Comment:
This workflow is intended to ensure that the current push is buildable. So replace the Dockerfile with the file you actually need to build the dev stage